File: Assignments/Assignment_65/program5.java
// write a program which accept number from user and ON its first 4 bits. Return modified number.

import java.util.Scanner;

class Bitwise
{
    public int ONBit(int iNo)
    {
        int iMask = 0;
        int iResult = 0;

        iMask = 0x0000000f;

        iResult = iNo | iMask;

        return iResult;
    }
}

class program5
{
    public static void main(String A[])
    { 
        int iValue = 0;
        int iRet = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Number : ");
        iValue = sobj.nextInt();

        Bitwise bobj = new Bitwise();
        iRet = bobj.ONBit(iValue);

        
        System.out.println("Modified number is : "+iRet);
    }
}

File: Assignments/Assignment_70/program5.c
// Write a recursive program which Accept Number from user and return its product of digits.
// Input  : 523
// Output : 30

#include<stdio.h>

int Mult(int iNo)
{
    static int iProd = 1;
    static int iDigit = 0;

    if(iNo != 0)
    {
        iDigit = iNo % 10;
        iProd = iProd * iDigit;
        iNo = iNo / 10;

        Mult(iNo);
    }

    return iProd;
}

int main()
{
    int iValue = 0, iRet = 0;

    printf("Enter Number : \n");
    scanf("%d",&iValue);

    iRet = Mult(iValue);

    printf("%d",iRet);

    return 0;
}
